### ISSUE TITLE
Building Objective-Git for accessing Github via SSH fails

### ISSUE THREAD

**Issue by ralfebert** [libgit2/objective-git#646]
I am currently trying to build the Objective-Git master branch to be able to connect to Github again via SSH in an iOS Xcode project as there seems to be no updated binary as of now.

When I try to build via Carthage, Cartfile:

```
github "libgit2/objective-git" "master"
```

via:

```
carthage update --platform iOS
```

I get:

```
xcodebuild: error: Scheme ObjectiveGit iOS is not currently configured for the archive action.
```

I also tried building via submodule / project embedded in Xcode as described in https://github.com/libgit2/objective-git#subproject , which seems to work fine, I also see OpenSSH+libssl being built, but at runtime I get an "unsupported URL protocol" error (I guess libgit2 doesn't pick up the ssh library?)

Any hints how to build and embed the updated Objective-Git currently in an iOS Xcode9 project?

**Comment by ralfebert** [comment]
Hmm, I got it to build using Carthage by manually changing the scheme and enabling the iOS framework for the archive build, but I still get the "unsupported URL protocol" error at runtime even though the SSH library was built. What's the trick to build Objective-Git with SSH support?

**Comment by ralfebert** [comment]
hmm, libssl and libssh2 were built successful, but `build-libgit2.log` says:

```
-- Checking for module 'libssh2'
--   Package 'libssl', required by 'libssh2', not found
-- LIBSSH2 not found. Set CMAKE_PREFIX_PATH if it is installed outside of the default search path.
```

**Comment by FireUponSky** [comment]
Greetings, anyone know how is this happening?

**Comment by tiennou** [comment]
We've just released [0.14.1](https://github.com/libgit2/objective-git/releases/tag/0.14.1), which should fix this issue, so I'm closing this. Feel free to reopen if applicable though.
